core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/shared/services/dot-experiments.service.ts

```diff
@@ -47,6 +47,18 @@ export class DotExperimentsService {
             .pipe(pluck('entity'));
     }
 
+    /**
+     * Get an array of experiments of a pageId that are running
+     * @param {string} pageId
+     * @returns Observable<DotExperiment[]>
+     * @memberof DotExperimentsService
+     */
+    getRunning(pageId: string): Observable<DotExperiment[]> {
```

Comment:
change to getByStatus and send pageId and a ExperimentStatus to do a little bit more generic.